components/scrapd-home-view/component.js

```diff
@@ -29,41 +34,40 @@ const ScrapdHomeView = () => {
   const grabData = [
     <div key="0">
       <p>
-        We are fully transparent with our data sets. We publish them on{' '}
+        The scrAPD web scraper generates data sets which populate our scrAPD dashboard. Use the “JSON” and “CSV” buttons
+        by the date picker to download a data set for a specified timeframe in the selected format.
+      </p>
+      <p>
+        The scrAPD team is committed to transparency. You can find our data sets published on{' '}
         <a href="https://github.com/scrapd/datasets/tree/master/datasets" target="_blank" rel="noopener noreferrer">
           GitHub
         </a>{' '}
         and{' '}
         <a href="https://data.world/remy-greinhofer/scrapd" target="_blank" rel="noopener noreferrer">
-          Data.World
+          Data.World.
         </a>{' '}
-        .
-      </p>
-      <p>
-        You can also use the buttons in the toolbar to retrieve a subset of data bounded by the dates you select in the
-        date picker. The available formats are JSON and CSV.
       </p>
     </div>
   ];
   const contribute = [
     <div key="0">
       <p>
-        All the ScrAPD projects are Open Source and hosted in the{' '}
-        <a href="https://github.com/scrapd">ScrAPD organization</a> on Github. We welcome curious and compasionate
-        collaborators, as well as meaningful contributions
+        The ScrAPD web scraper and dashboard are Open Source and hosted in the{' '}
+        <a href="https://github.com/scrapd">ScrAPD organization</a> on Github. We welcome curious, compassionate
+        collaborators to make meaningful contributions to the project{' '}
       </p>
+      {/* Maybe the below could be in a bulleted list */}
       <p>
         The general documentation is available <a href="https://docs.scrapd.org">here</a>.
       </p>
       <p>
         You can find the scraping tool we use to retrieve data at this{' '}
-        <a href="https://github.com/scrapd/scrapd"> link</a>. and the source code for this dashboard{' '}
+        <a href="https://github.com/scrapd/scrapd"> link</a>. and the source code for the dashboard{' '}
         <a href="https://github.com/scrapdviz/scrapdviz"> here</a>.
       </p>
       <p>
-        ScrAPD is a project developed mostly by vonlunteers from Open Austin, a local organization solving civic
-        problems using design and technology. You can learn more about it on their{' '}
-        <a href="https://open-austin.org">website</a>.
+        ScrAPD is a project developed mostly by volunteers from Open Austin, a local organization solving civic problems
```

Comment:
> ScrAPD is a project developed mostly by volunteers from Open Austin, a local organization solving civic problems

should be:

ScrAPD is a project developed by volunteers, mostly from Open Austin, a local organization solving civic problems

Reply:
Complete